bib/cmd/bootc-image-builder/mtls.go

```diff
@@ -19,6 +20,18 @@ type fileReader interface {
 	ReadFile(string) ([]byte, error)
 }
 
+type SimpleFileReader struct{}
+
+func (SimpleFileReader) ReadFile(path string) ([]byte, error) {
+	fp, err := os.Open(path)
```

Comment:
I think you can just use https://pkg.go.dev/os#ReadFile

Reply:
Indeed.  Silly me.  Amended.